collector/collector-build.sh

```diff
@@ -0,0 +1,13 @@
+#collector
+#cd kindling/collector
+
+# If developers start labeling later:
+# GitCommit=$(git describe)
+GitCommit=$(git rev-parse --short HEAD || echo unsupported)
+echo "Git commit:" $GitCommit
+
+
+docker run -it -v $PWD:/collector kindlingproject/kindling-collector-builder bash -c 'go build -o kindling-collector -ldflags="-X 'github.com/Kindling-project/kindling/collector/version.CodeVersion=$GitCommit'"'
```

Comment:
I think the file `collector-build.sh` should focus only on "building" and "injecting" git revision number. Anything about `docker` should move out of this file, in which way we could combine freely multiple scripts later. I think this is also one of the practices of the single responsibility principle.